<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(238, 236, 239);</string>
  </property>
  <widget class="QWidget" name="widget1" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>720</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#widget1 {background-color: rgb(147, 205, 255);};

</string>
   </property>
   <widget class="QPushButton" name="backButton">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>480</y>
      <width>100</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 500 15pt &quot;Product Sans&quot;;
border-radius: 10px;
background-color: rgb(255, 239, 115);


</string>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1000</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 239, 115);
color: rgb(0, 0, 0);
font: 75 40pt &quot;Product Sans&quot;;
font-weight: bold;
</string>
    </property>
    <property name="text">
     <string>Key Generator</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>111</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);
font: 15pt &quot;Product Sans&quot;;</string>
    </property>
    <property name="text">
     <string>n Key</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>121</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);
font: 15pt &quot;Product Sans&quot;;</string>
    </property>
    <property name="text">
     <string>d (private)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="loadEKey">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>300</y>
      <width>101</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 500 10pt &quot;Product Sans&quot;;
border-radius: 10px;
background-color: rgb(255, 239, 115);


</string>
    </property>
    <property name="text">
     <string>Load Key</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="eKey">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>300</y>
      <width>711</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="dKey">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>360</y>
      <width>711</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>111</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);
font: 15pt &quot;Product Sans&quot;;</string>
    </property>
    <property name="text">
     <string>e (public)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="loadDKey">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>360</y>
      <width>101</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 500 10pt &quot;Product Sans&quot;;
border-radius: 10px;
background-color: rgb(255, 239, 115);


</string>
    </property>
    <property name="text">
     <string>Load Key</string>
    </property>
   </widget>
   <widget class="QPushButton" name="generateKeyButton">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>410</y>
      <width>181</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 500 15pt &quot;Product Sans&quot;;
border-radius: 10px;
background-color: rgb(255, 239, 115);


</string>
    </property>
    <property name="text">
     <string>Generate Key</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveKeyButton">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>410</y>
      <width>131</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 500 15pt &quot;Product Sans&quot;;
border-radius: 10px;
background-color: rgb(255, 239, 115);


</string>
    </property>
    <property name="text">
     <string>Save Key</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="nKey">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>240</y>
      <width>821</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
